fava/static/sass/_journal-table.scss

```diff
@@ -244,6 +244,20 @@
     &.leg-pending { background-color: $color-journalentry-postingwarning; }
   }
 
+  .metadata-indicator {
+    font-size: 10px;
```

Comment:
There's only one other place where it occurs currently as far as I can see. In any case, there should never be more warnings after a commit really, only less, otherwise we'll never get rid of them.